src/uproot/containers.py

```diff
@@ -1441,16 +1439,17 @@ def awkward_form(self, file, context):
                     uproot._util.awkward_form(self._values, file, context),
                 ),
                 None,
+                parameters={"__array__": "sorted_map"},
             ),
             parameters={"__array__": "sorted_map"},
         )
```

Comment:
Only one level of this should be `"__array__": "sorted_map"`. I remember we were talking about this in one of our meetings, and thought about the consequences of both choices. Putting the `"__array__": "sorted_map"` in with the RecordForm, rather than out with the ListOffsetForm, allows for the possibility that a whole array can be one sorted map, something we'll likely want even if it's not what we have here. (Someone could take this `array` and do `array[0]` to get only one list of key-value pairs, and that single list should have map-like semantics someday.)

```suggestion
                    uproot._util.awkward_form(self._values, file, context),
                ),
                None,
                parameters={"__array__": "sorted_map"},
            ),
        )
```

Reply:
This might affect the test (src/uproot/streamers.py line 285). In which case, the expected test result can be changed again.